c_src/unifex/cnode/unifex/unifex.h

```diff
@@ -44,12 +45,35 @@ static inline void *unifex_alloc(size_t size) { return malloc(size); }
 
 static inline void unifex_free(void *pointer) { free(pointer); }
 
+/**
+ * Env
+ * 
+ * These functions are introduced to achieve interoperability between NIF and C node.
+ * They does nothing.
+*/
 static inline UnifexEnv *unifex_alloc_env(UnifexEnv *env) { return env; }
-
 static inline void unifex_clear_env(UnifexEnv *_env) { UNIFEX_UNUSED(_env); }
-
 static inline void unifex_free_env(UnifexEnv *_env) { UNIFEX_UNUSED(_env); }
 
+/** 
+ * Threads
+ * 
+ * To use these functions compile and link with -pthread.
```

Comment:
let's add that to bundlex configuration of unifex

Reply:
This way we will always link with `-pthread` even when someone doesn't need it. Are w okay with this?

Reply:
I think so, do you think that's a problem?

Reply:
Just a small overhead for memory usage. I will change it

Reply:
Done